gst/tensor_decoder/tensordec.c

```diff
@@ -172,6 +172,22 @@ static gboolean gst_tensordec_transform_size (GstBaseTransform * trans,
     GstPadDirection direction, GstCaps * caps, gsize size,
     GstCaps * othercaps, gsize * othersize);
 
+/** tensor decoder image labeling mode data info  */
+TensorDec_Mode_image_Label *tensordec_mode_image_label;
+
+/**
+ * @brief initialize data in tensor decoder image labeling info structure.
+ */
+static void
+gst_tensordec_image_labeling_init ()
+{
+  tensordec_mode_image_label->image_labeling_info.label_path = NULL;
```

Comment:
You are assigning values to an unallocated pointer, leading to possible Segmentation Fault.